cmd/manager/main.go

```diff
@@ -10,9 +10,9 @@ import (
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
-	"github.com/pmorie/kubefed-operator/pkg/apis"
-	"github.com/pmorie/kubefed-operator/pkg/controller"
-	operatorVersion "github.com/pmorie/kubefed-operator/version"
+	"github.com/openshift/kubefed-operator/pkg/apis"
```

Comment:
Don't the rest of the import paths need to change as well?

Reply:
haven't seen any issues with the smoke testing though.

Reply:
I have already changed it everywhere. Do you see any place where I missed it?

Reply:
well right now the trees haven't deviated much, so your smoke test will see very little impact.